src/steps/os/unix.rs

```diff
@@ -9,6 +9,7 @@ use std::process::Command;
 
 pub fn run_zplug(base_dirs: &BaseDirs, run_type: RunType) -> Result<(), Error> {
     let zsh = require("zsh")?;
+    print_separator("zplug");
```

Comment:
Oops. I probably forgot that.

Anyway, separators should be printed after all requirements have been made. The current implementation will print the zplug separator even if you only have zsh install and don't use zplug. You should move that line before the cmd line.

Reply:
You still need to remove this one